warehouse/query-core/src/main/java/datawave/query/jexl/visitors/ExpandCompositeTerms.java

```diff
@@ -173,9 +174,10 @@ public Object visit(ASTOrNode node, Object data) {
     public Object visit(ASTAndNode node, Object data) {
         ExpandData parentData = (ExpandData) data;
         
-        // ignore marked nodes
-        if (QueryPropertyMarkerVisitor.instanceOfAny(node))
+        // ignore marked nodes except delays
+        if (QueryPropertyMarkerVisitor.instanceOfAny(node, ASTDelayedPredicate.class)) {
```

Comment:
How about EvaluationOnly nodes?  Why only DelayedPredicates?

Reply:
I think you are right here. Looking back I'm not sure why EvaluationOnly would have been excluded here.